downloads.html

```diff
@@ -60,23 +60,66 @@ <h3 class="download-version">2.8.0<a href="#2.8.0"><i class="fas fa-link " style
         For more information, please read the detailed <a href="https://downloads.apache.org/kafka/2.8.0/RELEASE_NOTES.html">Release Notes</a>.
     </p>
 
+    <span id="2.7.1"></span>
+    <h3 class="download-version">2.7.1<a href="#2.7.1"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+    <ul>
+        <li>
+            Released May 10, 2020
+        </li>
+        <li>
+            <a href="https://downloads.apache.org/kafka/2.7.1/RELEASE_NOTES.html">Release Notes</a>
+        </li>
+        <li>
+            Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.7.1/kafka-2.7.1-src.tgz">kafka-2.7.1-src.tgz</a> (<a href="https://downloads.apache.org/kafka/2.7.1/kafka-2.7.1-src.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.7.1/kafka-2.7.1-src.tgz.sha512">sha512</a>)
+        </li>
+        <li>
+            Binary downloads:
+            <ul>
+                <li>Scala 2.12 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.7.1/kafka_2.12-2.7.1.tgz">kafka_2.12-2.7.1.tgz</a> (<a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.12-2.7.1.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.12-2.7.1.tgz.sha512">sha512</a>)</li>
+                <li>Scala 2.13 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.7.1/kafka_2.13-2.7.1.tgz">kafka_2.13-2.7.1.tgz</a> (<a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.13-2.7.1.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/2.7.1/kafka_2.13-2.7.1.tgz.sha512">sha512</a>)</li>
+            </ul>
+            We build for multiple versions of Scala. This only matters if you are using Scala and you want a version
+            built for the same Scala version you use. Otherwise any version should work (2.13 is recommended).
+        </li>
+    </ul>
+
+    <p>
+        Kafka 2.7.1 includes a number of significant new features. Here is a summary of some notable changes:
```

Comment:
Seems this list was copied from `2.7.0` release below, but does not really apply to a bug fix release? I think this paragraph and the bullet point list below should not be added, but only the last paragraph about `2.7.1`.